frameworks/hdfs/universe/resource.json

```diff
@@ -3,8 +3,10 @@
     "uris": {
       "jre-tar-gz": "{{jre-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
-      "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.11.0.tar.gz",
+      "keytab-zip": "https://downloads.mesosphere.com/hdfs/assets/krbtest-2.0.tar.gz",
+      "hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-3.2.0.tar.gz",
       "hdfs-bin-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hdfs-bin.tar.gz",
+      "hdfs-bug-issue-uri": "https://issues.apache.org/jira/browse/HADOOP-16283",
```

Comment:
`build.sh` failed with a error:
`Found a bad URI: issues.apache.org/jira/browse/HADOOP-16283" in: /build/frameworks/hdfs/src/main/dist/svc.yml Export URIs to resource.json to allow packaging for airgapped clusters.
Airgap check FAILED. This framework will NOT work in an airgap. Fix the detected issues.
`

Reply:
This resource is referenced in `resource.json` and `marathon.json.mustache`, but is not downloaded in sandbox, since it is not included in `-uris` in `svc`.

Reply:
Seems like we need to fix the airgap linter instead in this case, and still remove the entry added to the resources as Sam said.

Reply:
Ah I found the issue here. At first I thought, "that's strange. airgap_linter explicitly ignores comments: https://github.com/mesosphere/dcos-commons/blob/master/tools/airgap_linter.py#L33" but then I realized we're `echo`ing. In that case, let's change the echo to a comment. I just tried locally and `./build.sh aws` works as expected.

Reply:
Done